sources/packages/backend/apps/api/test/jest-e2e.json

```diff
@@ -25,5 +25,21 @@
     "@sims/test-utils": "<rootDir>/../../libs/test-utils/src",
     "@sims/integrations/(.*)": "<rootDir>/../../libs/integrations/src/$1",
     "@sims/integrations": "<rootDir>/../../libs/integrations/src"
-  }
+  },
+  "coveragePathIgnorePatterns": [
```

Comment:
As we talked about in the meeting, let's remove the non-API-related items.
Nice catch by the way 😉